packages/rx_bloc_cli/mason_templates/bricks/lib_change_language/__brick__/lib/lib_change_language/repositories/language_repository.dart

```diff
@@ -0,0 +1,22 @@
+{{> licence.dart }}
+
+import 'package:widget_toolkit/language_picker.dart';
+
+import '../data_sources/language_local_data_source.dart';
+
+class LanguageRepository {
+  LanguageRepository({
+    required LanguageLocalDataSource languageLocalDataSource,
+  }) : _languageLocalDataSource = languageLocalDataSource;
+
+  final LanguageLocalDataSource _languageLocalDataSource;
+
+  Future<List<LanguageModel>> getAll() async =>
```

Comment:
Please use the error mapper

Reply:
I have wraped the methods in `_errorMapper.execute()`